app/src/marketplace/components/Modal/ChooseAccessPeriodDialog/index.jsx

```diff
@@ -80,18 +88,42 @@ export const ChooseAccessPeriodDialog = ({
 
     const isValidTime = useCallback(() => !BN(time).isNaN() && BN(time).isGreaterThan(0), [time])
 
+    const isValidPrice = useCallback(() => {
+        if (paymentCurrency === paymentCurrencies.ETH) {
+            if (BN(priceInEth).isNaN() || !BN(priceInEth).isGreaterThan(0) || !BN(priceInEth).isFinite()) {
```

Comment:
Simply `return !(BN(priceInEth).isNaN() || !BN(priceInEth).isGreaterThan(0) || !BN(priceInEth).isFinite())`?

Reply:
Returning false is slightly more readable to me at a glance, but probably just a taste thing. Happy to tighten these up 👍

Reply:
both way is fine i think, readability is always good in code :)